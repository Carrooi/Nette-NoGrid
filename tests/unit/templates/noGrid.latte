<table n:no-grid="grid">
	<thead>
		<tr>
			<th>
				<ul>
					<li n:no-grid-views-as="$view" data-name="{$view->getName()}" n:class="$view->isCurrent() ? active">
						<a href="{$view->getLink()}">{$view->getTitle()}</a>
					</li>
				</ul>
			</th>
		</tr>
		<tr>
			<th>
				Number
			</th>
		</tr>
	</thead>
	<tbody>
		<tr n:no-grid-data-as="$line">
			<td>{$line[number]}</td>
		</tr>
		<tr n:no-grid-empty>
			<td>Empty</td>
		</tr>
	</tbody>
	<tfoot n:no-grid-has-paginator>
		<tr>
			<td>Paginator</td>
		</tr>
	</tfoot>
</table>
