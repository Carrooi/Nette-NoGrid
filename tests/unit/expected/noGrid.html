<table>
	<thead>
		<tr>
			<th>
				<ul>
					<li data-name="first">
						<a href="link">View 1</a>
					</li>
					<li data-name="second" class="active">
						<a href="link">View 2</a>
					</li>
				</ul>
			</th>
		</tr>
		<tr>
			<th>
				Number
			</th>
		</tr>
	</thead>
	<tbody>
		<tr>
			<td>1</td>
		</tr>
		<tr>
			<td>2</td>
		</tr>
		<tr>
			<td>3</td>
		</tr>
	</tbody>
</table>
